components/scream/tests/coupled/dynamics_physics/homme_shoc_cld_p3_rrtmgp/homme_shoc_cld_p3_rrtmgp_output.yaml

```diff
@@ -5,58 +5,58 @@ Averaging Type: Instant
 Max Snapshots Per File: 1
 Fields:
   Physics GLL:
-    - T_mid
-    - T_prev_micro_step
-    - qv
-    - qc
-    - qr
-    - qi
-    - qm
-    - nc
-    - nr
-    - ni
-    - bm
-    - qv_prev_micro_step
-    - eff_radius_qc
-    - eff_radius_qi
-    - micro_liq_ice_exchange
-    - micro_vap_liq_exchange
-    - micro_vap_ice_exchange
-    - tke
-    - cldfrac_liq
-    - cldfrac_tot
-    - eddy_diff_mom
-    - horiz_winds
-    - sgs_buoy_flux
-    - inv_qc_relvar
-    - pbl_height
-    - LW_flux_up
-    - LW_flux_dn
-    - SW_flux_up
-    - SW_flux_dn
-    - SW_flux_dn_dir
-    - horiz_winds_prev
-    - phi_int
-    - ps
-    - omega
-    - p_int
-    - p_mid
-    - phis
-    - pseudo_density
-    - surf_latent_flux
-    - surf_mom_flux
-    - surf_sens_flux
-    - cldfrac_ice
-    - nc_activated
-    - ni_activated
-    - nc_nuceat_tend
-    - eff_radius_qc
-    - eff_radius_qi
-    - sfc_alb_dir_nir
-    - sfc_alb_dir_vis
-    - sfc_alb_dif_nir
-    - sfc_alb_dif_vis
-    - surf_lw_flux_up
+    Fields Names:
```

Comment:
Does this change need to be made to the other uncoupled and coupled  tests?

Reply:
I thought I changed all the yaml files. But I now see I missed some, probably due to the rebase following your tests folder changing PR. I'll fix those.